client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.ts

```diff
@@ -16,11 +16,13 @@ export const convertDate = (dbDate: Date | null) => dbDate === null ? null : new
 const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDetailsOutcome => {
 
     const {
-        setSymptoms, setBackgroundDiseases, setIsolationCityName, setIsolationStreetName, setStreetsInCity, initialDBClinicalDetails, setInitialDBClinicalDetails
+        setSymptoms, setBackgroundDiseases, setIsolationCityName, setIsolationStreetName, setStreetsInCity, initialDBClinicalDetails,
+        setInitialDBClinicalDetails
     } = parameters;
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
     const userId = useSelector<StoreStateType, string>(state => state.user.id);
+    const address = useSelector<StoreStateType, DBAddress>(state => state.address);
```

Comment:
why do you use redux for address

Reply:
because I want to pass the address to the clinical details tab to initialize the isolation address